Makefile

```diff
@@ -3,7 +3,7 @@
 # ?= conditional assign, so users can pass options on the CLI instead of manually editing this file
 FLAGS?=
 
-pre-commit: checkrst
+pre-commit:
 	pre-commit run --all
```

Comment:
Follow up: add a flag for showing diff on failure.

Reply:
Addressed in #1181